packages/sql/src/index.ts

```diff
@@ -134,6 +134,10 @@ const SqlPlugin: Plugin<AST | string> = {
           value: 'spark',
           description: 'Spark: https://spark.apache.org',
         },
+        {
+          value: 'sqlite',
+          description: 'SQLite: https://www.sqlite.org/index.html',
```

Comment:
```suggestion
          description: 'SQLite: https://www.sqlite.org',
```

Personally, I'd prefer shorter links.